### ISSUE TITLE
Feature 8.4 - Implement model CreateGradientDescentStepsParams

### ISSUE THREAD

**Issue by borislav-vucicevic-04** [borislav-vucicevic-04/regressly#31]
Implement a model called **'CreateGradientDescentStepsParams'** that will contain parameters for a utility function, `create_gradient_descent_steps`, which will be implemented later.

The model will have the following:
- weights: list[float] - initial vector of weights
- dataset: list[list[float]] - the inputs of the dataset
- real_values: list[float] - list of real values
- predicted_values: list[float] - list of predicted values
- errors: list[float] - list of errors
- learning_rate: float - the learning rate
- gradient_components: list[list[float]] - list of gradient_components
- updated_weights: list[float] - vector of updated weights
- print_settings: PrintSettings - object containing page properties such as orientation, size and margin size.
